src/utils/number.spec.ts

```diff
@@ -113,17 +113,17 @@ describe("BigDecimal", () => {
 
   locales.forEach((locale) => {
     it(`correctly localizes number parts - ${locale}`, () => {
-      const parts = new BigDecimal("-12345678.9").formatToParts(locale);
+      numberStringFormatter.setOptions({
+        locale,
+        numberingSystem: "latn",
+        useGrouping: true
+      });
 
-      const group = getGroupSeparator(locale);
+      const parts = new BigDecimal("-12345678.9").formatToParts(numberStringFormatter.numberFormatter);
       const groupPart = parts.find((part) => part.type === "group").value;
-      expect(groupPart.trim().length === 0 ? " " : groupPart).toBe(group);
-
-      const decimal = getDecimalSeparator(locale);
-      expect(parts.find((part) => part.type === "decimal").value).toBe(decimal);
-
-      const minusSign = getMinusSign(locale);
-      expect(parts.find((part) => part.type === "minusSign").value).toBe(minusSign);
+      expect(groupPart.trim().length === 0 ? " " : groupPart).toBe(numberStringFormatter.group);
+      expect(parts.find((part) => part.type === "decimal").value).toBe(numberStringFormatter.decimal);
+      expect(parts.find((part) => part.type === "minusSign").value).toBe(numberStringFormatter.minusSign);
```

Comment:
Should I create getter methods instead of directly accessing these `NumberStringFormat` class properties?